pubspec.yaml

```diff
@@ -29,6 +29,10 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  domain:
```

Comment:
main yaml should have only presentation dependency. presentation already has domain

Reply:
done.

Reply:
done.